2023/AzureBeijing/data.json

```diff
@@ -0,0 +1,30 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "haoguanjun",
+      "",
+      ""
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "MSTechlnk",
+    "Country": "China",
+    "Url": "http://www.ngbeijing.cn/",
+    "SessionizeApiId": "{Your sessionize event Id}",
+    "Description_SessionizeApiId": "If you have configured your Sessionize correctly then browsing to this url (replacing {Your sessionize event Id} with your event sessionize id) will return valid json: https://sessionize.com/api/v2/{Your sessionize event Id}/view/Sessions",
+    "SessionizeEventName": "{Your sessionize event name (or complete URL to the public Sessionize C4P, i.e. https://sessionize.com/global-azure or just global-azure)}",
```

Comment:
You are not planning to synchronise your event sessions to the Global Azure web site so that all the world can see them? ;)

Reply:
Sorry, missing it, I just add them. Please help make it.